core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionUnitSuite.scala

```diff
@@ -88,5 +92,6 @@ class BlockManagerDecommissionUnitSuite extends SparkFunSuite with Matchers {
     } finally {
         bmDecomManager.stop()
     }
+    bmDecomManager.stop()
```

Comment:
Why do we need this ? The finally block above should run right ?

Reply:
good point.